lib/azkaban_scheduler/client.rb

```diff
@@ -19,24 +20,25 @@ def initialize(url, http_options = {})
     def get(path, params=nil, headers=nil)
```

Comment:
Personally I'd drop the headers param, no real need for it.

Reply:
some of the methods use them

Reply:
https://github.com/Shopify/azkaban-scheduler-ruby/blob/master/lib/azkaban_scheduler/session.rb#L152